gradle/dependencies.gradle

```diff
@@ -103,7 +103,7 @@ versions += [
   gradle: "8.5",
   grgit: "4.1.1",
   httpclient: "4.5.14",
-  jackson: "2.16.0",
+  jackson: "2.16.1",
```

Comment:
please add jackson diff to the description of the PR for completeness - https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.16.1 

I have reviewed the diff and I don't see any problems in this patch version.